README.md

```diff
@@ -62,74 +64,67 @@ support.
 ### How to install
 
 ```sh
-sudo apt-get install python3-venv
-
-pyvenv .venv
-. .venv/bin/activate
-
-# Stable version
-
-pip install homu
-
-# Development version
-
-git clone https://github.com/barosl/homu.git
-pip install -e homu
+$ git clone https://github.com/barosl/homu.git
+$ pip install -e homu
```

Comment:
The venv setup was already there in the README, this pull request removes it. Which we probably don't want.